Extra variables

OuFinx opened the pull request:
## Description

We want to use your module but we have figured out that you don't have some variables that we need. 

So, I've added these variables in the code and updated your README.md file. Also, tested it on my deployments and everything is OK.

Please check our feature and merge PR.

Thanks

soerenmartius commented:
> ## Description
> We want to use your module but we have figured out that you don't have some variables that we need.
> 
> So, I've added these variables in the code and updated your README.md file. Also, tested it on my deployments and everything is OK.
> 
> Please check our feature and merge PR.
> 
> Thanks

Hi @OuFinx , thanks for providing this. I believe we need to bump min required the version of the provider since these variables have been implemented in a newer version. Let me check this with the team and come back to you in a little while.

samen93 reviewed (commented):
looks good to me

Hanymin reviewed (commented):
lgtm

OuFinx commented:
> > ## Description
> > We want to use your module but we have figured out that you don't have some variables that we need.
> > So, I've added these variables in the code and updated your README.md file. Also, tested it on my deployments and everything is OK.
> > Please check our feature and merge PR.
> > Thanks
> 
> Hi @OuFinx , thanks for providing this. I believe we need to bump min required the version of the provider since these variables have been implemented in a newer version. Let me check this with the team and come back to you in a little while.

Hi, what we will do with this PR?

mariux commented:
closingin favor of https://github.com/mineiros-io/terraform-aws-cognito-user-pool/pull/51
